src/WebJobs.Extensions.DurableTask/Correlation/TraceHelper.cs

```diff
@@ -60,7 +61,7 @@ internal class TraceHelper
                 Schema.SpanNames.CallOrSignalEntity(entityName, operationName),
                 kind: signalEntity ? ActivityKind.Producer : ActivityKind.Client,
                 parentContext: parentTraceContext.Value,
-                startTime: startTime);
+                startTime: startTime ?? default);

```

Comment:
Can we remove this change since we added the default value in the `startTime` optional parameter?

Reply:
Unfortunately no, the issue is that I had to make the startTime parameter a `DateTimeOffset?` rather than a `DateTimeOffset` even with the default value because of how this method is called in the [LocalGrpcListener](https://github.com/Azure/azure-functions-durable-extension/blob/ea9f27b4e3d5974eee9eb010015f3b7e57e6ff43/src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs#L189) (the issue being that we could get null when trying to extract the request time from the `CreateInstanceRequest`) and so that's why I had to add this change to the `TraceHelper` method